test/stdlib/TestData.swift

```diff
@@ -739,7 +739,7 @@ class TestData : TestDataSuper {
         data.append("test", count: 4)
         expectTrue(object.verifier.wasMutableCopied)
 
-        let preservedObjectness = (data as NSData) is MutableDataVerifier
+        let preservedObjectness = (data as NSData) is ImmutableDataVerifier
```

Comment:
this should never have been mutable since it would leak out potentially unsafe shared changes out from under the storage